pyartifactory/models/auth.py

```diff
@@ -12,7 +12,7 @@ class AuthModel(BaseModel):
 
     url: str
     auth: Tuple[str, SecretStr]
-    verify: bool = True
+    verify: bool | str = True

```

Comment:
this syntax does not work on Python < 3.9

Reply:
Oh! my bad. It should be fixed by now.